### PR TITLE
 CI with check migrations

### PR DESCRIPTION
**Pull request by yogaxpto**
add CI step to ensure that all migration operations are created.

### CODE REVIEW

File: test_ci_backend.sh [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+# run django migrations check to ensure that there are no migrations left to create
+cd src || exit 1
```
Comment (lfrodrigues) [inline test_ci_backend.sh]: move to after line 20 (where you already did a cd)

Comment (yogaxpto) [reply test_ci_backend.sh]: Changed as requested.

**Comment by yogaxpto** [comment]
@fgarci03 please review if possible.

**Comment by yogaxpto** [comment]
Closed as this is not needed anymore. 
@fgarci03

**Pull request closed without merging**